app/tr_php/01/task05.php

```diff
@@ -7,10 +7,57 @@
     'よくわかるPHPの教科書',
 ];
 
-$input;
-$input_text;
-$result;
+$input=[];
+$input_text='';
+$result='';
```

Comment:
規約の確認をお願いします。
・演算子の前後は半角スペースが必要です。
```suggestion
$input = [];
$input_text = '';
$result = '';
```

Reply:
ご指摘、ありがとうございます！
演算子の前後は半角スペースが必要であるとのこと、理解しました。
これに関連する箇所を修正いたしました。